PiwikTracker/PiwikTracker.swift

```diff
@@ -260,13 +216,15 @@ extension PiwikTracker {
     ///
     /// - Parameter view: An array of hierarchical screen names.
     /// - Parameter url: The url of the page that was viewed. If none set the url will be http://example.com appended by the screen segments. Example: http://example.com/players/john-appleseed
```

Comment:
The documentation here should mention the `dimensions` parameter.

Reply:
Good catch! Done